doc/releases/octopus.rst

```diff
@@ -1194,7 +1303,7 @@ Major Changes from Nautilus
 
 General
 ~~~~~~~
-  
+
```

Comment:
Nit: I would prefer no unrelated whitespace changes or, if it is really irritating and makes your editor go crazy, as a separate commit.